src/main/java/core/basesyntax/service/impl/DataConverterServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverterService;
+import java.util.List;
+
+public class DataConverterServiceImpl implements DataConverterService {
+    private static final int TITTLE_POSITION = 1;
```

Comment:
The constant name `TITTLE_POSITION` seems to be a typo. Consider renaming it to `TITLE_POSITION` for correct spelling.